### ISSUE TITLE
Does not work with HA 2023.8.0

### ISSUE THREAD

**Issue by catsmanac** [catsmanac/Envoy_2_Core_custom_test#3]
Setup failed for custom integration enphase_envoy: Requirements for enphase_envoy not found: ['pyjwt==2.7.0'].

**Comment by catsmanac** [comment]
As this is formatted as custom integration remove versions from requirements so it can run with old and new HA version.

```
  "requirements": [
    "pyjwt",
    "xmltodict",
    "httpx",
    "envoy_utils"
  ],
````

**Comment by catsmanac** [comment]
Fix provided in release 0.2.8. Update in HACS.

**Comment by DMedina559** [comment]
That worked
